ClientApp/package.json

```diff
@@ -4,66 +4,55 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
-    "@types/jest": "^24.0.23",
-    "@types/moment-timezone": "^0.5.13",
-    "@types/node": "^12.12.16",
-    "@types/react": "^16.9.16",
+    "@types/node": "^16.18.10",
+    "@types/react": "^18.0.26",
     "@types/react-csv": "^1.1.1",
     "@types/react-datepicker": "^2.11.0",
-    "@types/react-dom": "^16.9.4",
+    "@types/react-dom": "^18.0.9",
     "@types/react-redux": "^7.1.7",
     "@types/react-router": "^5.1.5",
     "@types/react-router-dom": "^5.1.3",
     "@types/react-select": "^3.0.13",
     "@types/react-table": "^7.0.16",
-    "@typescript-eslint/eslint-plugin": "^2.11.0",
-    "@typescript-eslint/parser": "^2.11.0",
+    "@types/redux": "^3.6.0",
     "bootstrap": "^4.4.1",
-    "class-transformer": "^0.3.1",
+    "class-transformer": "^0.5.1",
     "client-oauth2": "^4.2.5",
     "email-validator": "^2.0.4",
-    "eslint-config-prettier": "^6.7.0",
-    "eslint-config-react": "^1.1.7",
-    "eslint-plugin-prettier": "^3.1.1",
+    "gh-pages": "^4.0.0",
```

Comment:
Is this being used?

Reply:
Oops, no, good catch. Copied from a newly-generated Create React App template a bit too hastily!